bin/plugin/commands/performance.js

```diff
@@ -30,54 +30,57 @@ const config = require( '../config' );
- * @property {number[]} firstPaint           Represents the time when the user agent first rendered after navigation.
- * @property {number[]} domContentLoaded     Represents the time immediately after the document's DOMContentLoaded event completes.
- * @property {number[]} loaded               Represents the time when the load event of the current document is completed.
- * @property {number[]} firstContentfulPaint Represents the time when the browser first renders any text or media.
- * @property {number[]} firstBlock           Represents the time when Puppeteer first sees a block selector in the DOM.
- * @property {number[]} type                 Average type time.
- * @property {number[]} typeContainer        Average type time within a container.
- * @property {number[]} focus                Average block selection time.
- * @property {number[]} inserterOpen         Average time to open global inserter.
- * @property {number[]} inserterSearch       Average time to search the inserter.
- * @property {number[]} inserterHover        Average time to move mouse between two block item in the inserter.
- * @property {number[]} listViewOpen         Average time to open listView
+ * @property {number[]} timeToFirstByte        Represents the time since the browser started the request until it received a response.
+ * @property {number[]} largestContentfulPaint Represents the time when the main content of the page has likely loaded.
+ * @property {number[]} serverResponse         Represents the time the server takes to respond.
+ * @property {number[]} firstPaint             Represents the time when the user agent first rendered after navigation.
+ * @property {number[]} domContentLoaded       Represents the time immediately after the document's DOMContentLoaded event completes.
+ * @property {number[]} loaded                 Represents the time when the load event of the current document is completed.
+ * @property {number[]} firstContentfulPaint   Represents the time when the browser first renders any text or media.
+ * @property {number[]} firstBlock             Represents the time when Puppeteer first sees a block selector in the DOM.
+ * @property {number[]} type                   Average type time.
+ * @property {number[]} typeContainer          Average type time within a container.
+ * @property {number[]} focus                  Average block selection time.
+ * @property {number[]} inserterOpen           Average time to open global inserter.
+ * @property {number[]} inserterSearch         Average time to search the inserter.
+ * @property {number[]} inserterHover          Average time to move mouse between two block item in the inserter.
+ * @property {number[]} listViewOpen           Average time to open listView
  */
 
 /**
  * @typedef WPPerformanceResults
  *
- * @property {number=} timeToFirstByteMedian Represents the time since the browser started the request until it received a response (median).
```

Comment:
This just adds two other properties: largestContentfulPaintMedian and largestContentfulPaintP75. The rest is indentation.